packages/v4/src/content/design-guidelines/styles/icons/icons.md

```diff
@@ -110,6 +110,14 @@ Use this for 'fa' regular icons: `<i className="far [insert-icon-name]"></i>`<br
 If you're looking to copy React for an icon:<br />
```

Comment:
I can't comment on the lines above this, but they reference:

`<i className="pf-icon [insert-icon-name]"></i>`

Seems like that is a mix of HTML and JSX? Wouldn't it either be `<i class="pf-icon [insert-icon-name]"></i>` or `<i className="pf-icon [insert-icon-name]" />`?  Since we're referencing writing HTML, I would probably write it as `<i class="pf-icon [insert-icon-name]"></i>` cc @tlabaj

Reply:
I made the other updates - just waiting on Titani for this one as I can't advise!

Reply:
This looks great! I'd like to make the use of `class` and `className` consistent though. I think using `class` in place of `className` on this page should be fine, we write regular HTML in the [a11y guide page](https://patternfly-org-pr-2475-v4.surge.sh/v4/developer-resources/accessibility-guide), too. wdty @redallen? Would love a second opinion!

Reply:
`class` is for HTML and `className` is for React.

Reply:
@mcoker fixed class name if you want to give this another look!